projects/angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -50,132 +34,59 @@
   .clr-focus-indicator {
-    @include mixins.css-var(
-      background-color,
-      clr-forms-focused-color,
-      forms-variables.$clr-forms-focused-color,
-      variables.$clr-use-custom-properties
-    );
-    height: variables.$clr_baselineRem_2px;
+    background-color: forms-variables.$clr-forms-focused-color;
+    height: tokens.$cds-global-space-2;
     width: 0;
     transition: width 0.2s ease;
     position: absolute;
-    bottom: -(variables.$clr_baselineRem_1px);
+    bottom: calc(-1 * #{tokens.$cds-global-space-1});
     left: 0;
-    @include mixins.css-var(
-      border-bottom-left-radius,
-      clr-combobox-border-radius,
-      combobox-variables.$clr-combobox-border-radius,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-bottom-right-radius,
-      clr-combobox-border-radius,
-      combobox-variables.$clr-combobox-border-radius,
-      variables.$clr-use-custom-properties
-    );
+    border-bottom-left-radius: combobox-variables.$clr-combobox-border-radius;
+    border-bottom-right-radius: combobox-variables.$clr-combobox-border-radius;
   }
   .clr-focus {
     width: 100%;
   }
   .clr-error .clr-focus-indicator {
-    @include mixins.css-var(
-      background-color,
-      clr-forms-invalid-color,
-      forms-variables.$clr-forms-invalid-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: forms-variables.$clr-forms-invalid-color;
   }
 
   .clr-combobox-wrapper {
     position: relative;
 
     //Dimensions
-    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-left-padding;
-    @include mixins.css-var(
-      padding-left,
-      clr-combobox-padding-left,
-      combobox-variables.$clr-combobox-trigger-left-padding,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      min-height,
-      clr-combobox-min-height,
-      combobox-variables.$clr-combobox-min-height,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      min-width,
-      clr-combobox-min-width,
-      combobox-variables.$clr-combobox-min-width,
-      variables.$clr-use-custom-properties
-    );
+    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-padding-left;
+    padding-left: combobox-variables.$clr-combobox-padding-left;
+
+    min-height: combobox-variables.$clr-combobox-min-height;
+    min-width: combobox-variables.$clr-combobox-min-width;
     .clr-input {
-      padding-right: 0;
-      @include mixins.css-var(
-        padding-left,
-        clr-combobox-input-left,
-        variables.$clr_baselineRem_0_25,
-        variables.$clr-use-custom-properties
-      );
-      // ATM remove
-      @include mixins.css-var(
-        max-height,
-        clr-combobox-input-height,
-        variables.$clr_baselineRem_1_25,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
+      max-height: tokens.$cds-global-space-7;
```

Comment:
ATM comment states to remove `max-height`. Why do we keep it?

Reply:
The comment was too short.
It refers to the refactor stage of the ATM. Can't remove it now - it will increase the default height of the combobox.
If it gets removed or kept will depend on the implementation once it's done.